NitroxClient/GameLogic/Vehicles.cs

```diff
@@ -200,7 +244,9 @@ private void OnVehiclePrefabLoaded(TechType techType, GameObject prefab, NitroxI
             }
 
             VehicleChildObjectIdentifierHelper.SetInteractiveChildrenIds(gameObject, interactiveChildIdentifiers); //Copy From ConstructorBeginCraftingProcessor
-                  
+            
+            Log.Debug($"Successfully created vehicle {id}");
```

Comment:
```suggestion
```

Success debug logs don't add much value. Only add logs that tell us something is going wrong.